velox/substrait/TypeUtils.h

```diff
@@ -21,4 +21,10 @@ namespace facebook::velox::substrait {
 /// Return the Velox type according to the typename.
 TypePtr toVeloxType(const std::string& typeName);
 
+std::vector<std::string_view> getTypesFromCompoundName(
```

Comment:
Does this function need to be in the header?

Reply:
I think it is not necessary, so I removed this code.